callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsConfigProvider.java

```diff
@@ -0,0 +1,70 @@
+package io.callstats.sdk.internal;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class CallStatsConfigProvider {
+
+  /** The Constant logger. */
+  private static final Logger logger = LogManager.getLogger("CallStatsConfigProvider");
+
+
+  /** The base url. */
+  public static String EVENTS_BASE_URL;
+  public static String STATS_BASE_URL;
+  public static String AUTH_BASE_URL;
+
+
+  /** The connection time out. */
+  public static int connectionTimeOut = CallStatsConst.CONNECTION_TIMEOUT;
+
+  public static void init() {
+    Properties prop = new Properties();
+    InputStream input = null;
+
+    try {
+      input = new FileInputStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
```

Comment:
We are reading this in quite a few places, would it be better to move at least reading to single place?

Reply:
Yes I removed in other places and created this new class.

Reply:
It's still done here: https://github.com/callstats-io/callstats.java/blob/b7b5134798c3a96ec40bc27817cf685015314960/callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsBridgeKeepAliveManager.java#L80